apps/web/src/store/modules/file/actions.ts

```diff
@@ -116,3 +116,15 @@ export const downloadExcel: Action<FileState, any> = async ({ commit, rootState,
         dispatch('display/finishLoading', {}, { root: true });
     }
 };
+
+export const downloadByFileUrl: Action<ExportResponse, any> = async (_, downloadUrl): Promise<void> => {
```

Comment:
issue: `ExportResponse` you declared is an object but the payload(the second argument which is named with 'downloadUrl') seems just `string` type.
It seems that `downloadUrl` is wrong for the second argument for this action.

```typescript
export const downloadByFileUrl: Action<ExportResponse, any> = async (_,{ downloadUrl }): 
```

I think it would be right.

Reply:
I don't understand why the second argument is wrong. plz explain more.

Reply:
I don't understand why the second argument is wrong. plz explain more.

Reply:
I don't think I fully understand the concept of the store and how to use it.